designcompose/src/main/java/com/android/designcompose/Layout.kt

```diff
@@ -132,12 +132,15 @@ internal object LayoutManager {
         name: String,
         textMeasureData: TextMeasureData,
     ) {
-        textMeasures[layoutId] = textMeasureData
-        subscribe(layoutId, setLayoutState, parentLayoutId, childIndex, style, name, true)
-
-        // Text cannot have children, so call computeLayoutIfComplete() here so that if this is the
-        // text or text style changed when no other nodes changed, we recompute layout
-        computeLayoutIfComplete(layoutId, rootLayoutId)
+        trace(DCTraces.LAYOUTMANAGER_SUBSCRIBEWITHMEASURE) {
+            textMeasures[layoutId] = textMeasureData
+            subscribe(layoutId, setLayoutState, parentLayoutId, childIndex, style, name, true)
+
+            // Text cannot have children, so call computeLayoutIfComplete() here so that if this is
+            // the
+            // text or text style changed when no other nodes changed, we recompute layout
+            computeLayoutIfComplete(layoutId, rootLayoutId)
```

Comment:
This function asks Rust to compute layout, so it could be worth tracing.

Reply:
It's wrapped in the LAYOUTMANAGER_SUBSCRIBEWITHMEASURE trace

Reply:
Yes, but it's called in other places also, in particular from `DesignFrame`.

Reply:
Ok, moved the trace from subscribe with measure to computeLayout.